### DISCUSSION TITLE
Deploying with another frontend

### DISCUSSION THREAD

**Discussion by blnkoff** [mkdocs/mkdocs#3843] [Q&A]
I need to create and deploy docs on my site. I want to use `MkDocs` with the material theme. 
I have developed and deploy a single-page site using Svelte.

I have the domain `domain.com`. It is hosted on cloudflare.com and frontend have to be accessible at `domain.com`. I want to deploy docs so that, it was accessible using the path `/docs`, that is `domain.com/docs`. Or if it is impossible, using the subdomain `docs`, that is `docs.domain.com`. Following the first case is preferable for me.

**Comment by thomas-mc-work** [comment]
I've created a demo project as a minimal showcase on how to integrate Svelte into a MkDocs project:

https://codeberg.org/common-good/mkdocs-svelte-integration

Maybe this is helpful to you.
